docs/install.md

```diff
@@ -230,6 +230,24 @@ SHARP_IGNORE_GLOBAL_LIBVIPS=1 npm install --arch=x64 --platform=linux --libc=gli
 To get the best performance select the largest memory available.
 A 1536 MB function provides ~12x more CPU time than a 128 MB function.
 
+### serverless-esbuild
+
+To deploy AWS Lambda using serverless framework, esbuild and serverless-esbuild on machines other than Linux x64 (glibc), first ensure sharp is excluded from bundling via the
+[exclude option](https://www.serverless.com/plugins/serverless-esbuild#options)
```

Comment:
Should this say `external` rather than `exclude`?

Reply:
I presume external makes more logical than exclude. Changing it to external.